setup.cfg

```diff
@@ -43,16 +43,19 @@ package_dir =
 # Version specifiers like >=2.2,<3.0 avoid problems due to API changes in
 # new major versions. This works if the required packages follow Semantic Versioning.
 # For more information, check out https://semver.org/.
+# Note: pyparsing is actually needed by QKeras, but missing as dependency
+# tf2onnx 1.9.3 first version after fixing https://github.com/onnx/tensorflow-onnx/issues/1707
 install_requires =
     importlib-metadata; python_version<"3.8"
     clize==4.1.1
     protobuf==3.20.1
     bitstring>=3.1.7
-    numpy
+    numpy==1.24.1
```

Comment:
@selwyn96 since the `np.bool` vs `bool` issue is fixed in `tf2onnx 1.12.1`, can we remove this harsh pin?

Reply:
There is a conflict with the main branch that pins numpy to 1.24.1

Reply:
Ah I see. I didn't realize this was introduced by @maltanar already. What is the reason again?